docs/custom_service.md

```diff
@@ -106,6 +106,8 @@ def handle(data, context):
 Here the ``` handle()``` method is our entry point that will be invoked by MMS, with the parameters data and context, it in turn can pass this information to an actual inference class object or handle all the processing in the 
 ```handle()``` method itself. The ```initialize()``` method is used to initialize the model at load time, so after first time, the service need not be re-initialized in the the life cycle of the relevant worker.
  We recommend using a ```initialize()``` method, avoid initialization at prediction time.
+ 
+ This entry point is engaged in two cases: (1) when MMS gets a ```PUT /models``` request or (2) when MMS gets ```POST /predictions``` request. (1) is used to scale up workers for model that has been already registered, e.g. scale model up. (2) is used as standard way to run inference against models. (1) is also known as model load time, and that is where you would normally want to put code for model initialization. _Note_: (1) will also be engaged if during MMS startup (```mxnet-model-server --start```) you provide model(s) to load OR if when MMS receives a ```POST /models``` call with `initial-workers` option as that call is used to update model state within MMS and could result in adjustment of scale for the model, e.g. decrease or increase amount of its workers. You can find out mode about these and other MMS APIs in [MMS Management API](./management_api.md).
```

Comment:
These are not shown as points.

Reply:
I meant them to not be shown as points, I just wanted to enumerate them so I can refer to them by indices rather than having to re-spell in words over and over.